### PR TITLE
add comment for interpolate_boundary_values

### PR DESCRIPTION
**Pull request by Irabiel**

### CODE REVIEW

**Comment by tjhei** [comment]
Irabiel, thanks for your PR. Sadly, your editor changed a lot of the formatting (look at the diff under "files changes"). Can you please run ``./contrib/utilities/indent`` in your deal.II folder and commit the result? If you need help with that, grab one of the developers.

**Comment by Irabiel** [comment]
Hi @tjhei we fixed the issue. Thanks to @drwells :)

File: examples/step-4/step-4.cc [hunk 1]
```diff
@@ -410,12 +410,20 @@ void Step4<dim>::assemble_system()
...
   // object of the class which describes the boundary values we would like to
   // use (i.e. the <code>BoundaryValues</code> class declared above):
+  //
+  // The function VectorTools::interpolate_boundary_values() will only work
+  // on faces that have been marked with boundary indicator 0 (because that's
+  // what we say the function should work on with the second argument below).
+  // If there are faces with boundary id other than 0, then the function
+  // interpolate_boundary_values will do nothing on these faces. For
+  // the Laplace equation doing nothing is equivalent to assuming that
+  // on those parts of the boundary, zero Neumann
```
Comment (drwells) [inline examples/step-4/step-4.cc]:
```suggestion
  // on those parts of the boundary, a zero Neumann
```

File: examples/step-4/step-4.cc [hunk 2]
```diff
@@ -410,12 +410,20 @@ void Step4<dim>::assemble_system()
...
+  // boundary condition hold.
```
Comment (drwells) [inline examples/step-4/step-4.cc]:
```suggestion
  // boundary condition holds.
```

**Comment by drwells** [comment]
@Irabiel We still need to fix the two minor grammar problems. Would it be okay with you if I created a new branch with your commit as well as a followup commit to get this done?

**Comment by drwells** [comment]
It was quick so I went ahead and made the suggested changes myself: its just three characters :)

Closed in favor of the collated PR #8642.

**Pull request closed without merging**